### DISCUSSION TITLE
how to autoreload ssl file

### DISCUSSION THREAD

**Discussion by leon-thomm** [Kludex/uvicorn#1511] [Q&A]
I'm using FastAPI for a website and I'm starting my webserver like this

```
uvicorn --host 0.0.0.0 --port 443 --ssl-keyfile=/etc/letsencrypt/live/<website>/privkey.pem --ssl-certfile /etc/letsencrypt/live/<website>/fullchain.pem main:app
```

and I ran into the issue of the updated ssl file not being reloaded by the server, it seems to cache it. How can auto-reload the SSL file, or at least reload it without restarting the uvicorn server?

Thanks!

**Comment by euri10** [comment]
I dont think that would be possible
